src/main/java/ua/lviv/iot/mylightstore/manager/StoreManager.java

```diff
@@ -0,0 +1,89 @@
+package ua.lviv.iot.mylightstore.manager;
+
+import ua.lviv.iot.mylightstore.model.Category;
+import ua.lviv.iot.mylightstore.model.SortDirection;
+import ua.lviv.iot.mylightstore.model.Store;
+import ua.lviv.iot.mylightstore.model.item.impl.Item;
+
+import java.util.*;
```

Comment:
Not an issue, it won't mess up your code, but just a recommendation for the future: better not import the whole package, just the things you need from it, e.g. Comparator.
Reasoning: there _MIGHT_ be class collisions in imports. There might be a situation where you'll * from two packages and they both might have classes named the same, and you won't be sure which one you're using. For example, you'll import * from _util_ and * from _sql_ packages, and they both have Date class.

Reply:
fixed